src/components/notifications/Slack.vue

```diff
@@ -9,6 +9,12 @@
         <label for="slack-channel" class="form-label">{{ $t("Channel Name") }}</label>
         <input id="slack-channel-name" v-model="$parent.notification.slackchannel" type="text" class="form-control">
 
+        <label class="form-label">{{ $t("Message format") }}</label>
+        <div class="form-check form-switch">
+            <input id="slack-text-message" v-model="$parent.notification.slackrichmessage" type="checkbox" class="form-check-input">
+            <label for="slack-text-message" class="form-label">{{ $t("Send rich messages") }}</label>
```

Comment:
Please add the new translaton keys `Message format` and `Send rich messages` to [`en.json`](https://github.com/louislam/uptime-kuma/blob/master/src/lang/en.json).

Without these keys being there, our translators won't be able to translate this string in weblate ^^

Reply:
Done!